src/clientManager.js

```diff
@@ -34,7 +34,11 @@ ClientManager.prototype.init = function (identHandler, sockets) {
 };
 
 ClientManager.prototype.findClient = function (name) {
-	return this.clients.find((u) => u.name === name);
+	// return this.clients.find((u) => u.name === name);
```

Comment:
Remove this comment.

It also would be nice to move `name = name.toLowerCase()` here, as there's no reason to lowercase the input more than once.

Reply:
Can you please have a look once again, I have modified the code after your comment

Reply:
My comment still applies.